### DISCUSSION TITLE
Helm chart support for storage options.

### DISCUSSION THREAD

**Discussion by mp365** [dexidp/dex#2251] [Q&A] [answered]
Hello,

We've been using the Helm chart to install and manage Dex on our K8s cluster (v1.19 EKS). To persist state we use K8s CRDs. We would like to change this to use a Postrgresql back end. 

Can the [Helm chart ](https://github.com/dexidp/helm-charts) be used to configure the storage options? Does it support configuration of the Dex storage options? 

I'm pretty new to both Helm and Dex so might have missed something obvious regarding this. I been looking at the Helm chart confg and docs but have not yet found values that can be used to configure the storage options. We're using an older version of the chart (0.1.1) so need to update that as well. 

Any help understanding if and how to configure the storage options using Helm would be greatly appreciated. 

Thanks
Mark

**Comment by mp365** [comment]
The list of Values for the chart [here ](https://artifacthub.io/packages/helm/dex/dex/0.6.0#values)doesn't include an entry for the 'storage' key. But I've noticed that the example at the top of the page includes the following: 
```
  # See https://dexidp.io/docs/storage/ for more options
  storage:
    type: memory
```
I think this probably answers the original question. I will use this along with the postgresql options documented [here](https://dexidp.io/docs/storage/#postgres).

**Comment by sagikazarmark** [comment, marked as answer]
There is a `config` key that allows adding any Dex configuration to the values file. As shown in the example that you also referenced.

**Reply by ger1964jou** [reply]
> There is a `config` key that allows adding any Dex configuration to the values file. As shown in the example that you also referenced.
